### PR TITLE
Narrowed the scope of the windows_counter_refresh_interval

### PR DESCRIPTION
**Pull request by iglendd**
Relects changes to the integrations-core
(https://github.com/DataDog/integrations-core/pull/13243)

### Reviewer's Checklist

- [ ] If known, an appropriate milestone has been selected; otherwise the `Triage` milestone is set.
- [ ] Use the `major_change` label if your change either has a major impact on the code base, is impacting multiple teams or is changing important well-established internals of the Agent. This label will be use during QA to make sure each team pay extra attention to the changed behavior. For any customer facing change use a releasenote.
- [ ] A [release note](https://github.com/DataDog/datadog-agent/blob/main/docs/dev/contributing.md#reno) has been added or the `changelog/no-changelog` label has been applied.
- [ ] Changed code has automated tests for its functionality.
- [ ] Adequate QA/testing plan information is provided if the `qa/skip-qa` label is not applied.
- [ ] At least one `team/..` label has been applied, indicating the team(s) that should QA this change.
- [ ] If applicable, docs team has been notified or [an issue has been opened on the documentation repo](https://github.com/DataDog/documentation/issues/new).
- [ ] If applicable, the `need-change/operator` and `need-change/helm` labels have been applied.
- [ ] If applicable, the `k8s/<min-version>` label, indicating the lowest Kubernetes version compatible with this feature. 
- [ ] If applicable, the [config template](https://github.com/DataDog/datadog-agent/blob/main/pkg/config/config_template.yaml) has been updated.

### CODE REVIEW

File: pkg/config/config_template.yaml [hunk 1]
```diff
@@ -897,11 +897,10 @@ api_key:
 
 {{- if (eq .OS "windows")}}
   ## @param windows_counter_refresh_interval - number - optional - default: 60
-  ## This controls the interval in seconds for which windows performance counters are refreshed by the agent.
-  ## Refreshes are necessary for the discovery of new counters/countersets and instances for integrations that
-  ## rely on performance counters. Checks include active_directory, aspdotnet, dotnetclr, exchange_server, hyperv,
-  ## iis and windows_performance_counters.
-  ## Refresh can be disabled by setting the interval to 0.
+  ## This controls frequency in seconds of refreshing cached performance counters objects list. It maybe
```
Comment (alai97) [inline pkg/config/config_template.yaml:900]:
```suggestion
  ## This controls frequency in seconds for refreshing the cached performance counters objects list. It maybe
```

File: pkg/config/config_template.yaml [hunk 2]
```diff
@@ -897,11 +897,10 @@ api_key:
...
+  ## useful in rare cases when initialization of a few Agent’s builtin performance counters based checks
```
Comment (alai97) [inline pkg/config/config_template.yaml:901]:
```suggestion
  ## useful in rare cases when initialization of some of the Agent’s builtin performance counters-based checks
```

**Review by alai97 — APPROVED** [review]
Looks good pending two copy suggestions!

**Review by sgnn7 — APPROVED** [review]

File: pkg/config/config_template.yaml (hunk 2 above)
Comment (iglendd) [reply pkg/config/config_template.yaml:901]: Thank you @sgnn7 and @alai97 for the review. After deliberation with @clarkb7 we have decided to remove its documentation from config template entirely (but not from the code) and leave a comment of similar sort in the code and new PR,

Comment (iglendd) [reply pkg/config/config_template.yaml:901]: Closing the PR to create a new one.

**Pull request closed without merging**